internal/config/config.cue

```diff
@@ -0,0 +1,5 @@
+{
+    data_dir: string
+    schema_dir: string | *"schemas"
```

Comment:
Should be schema. Is schemas a word!? I need to stop typing that

Reply:
whatever we decide, it should be consistent.